JumpscaleCore/clients/test/test_sonic_client.py

```diff
@@ -0,0 +1,318 @@
+import unittest
+from Jumpscale import j
+from random import randint
+from base_test import BaseTest
+
+
+@unittest.skip("https://github.com/threefoldtech/jumpscaleX_core/issues/250")
+class SonicClient(BaseTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.info("install sonic builder")
+        j.builders.apps.sonic.install()
+
+        cls.info("Start Sonic server")
+        j.servers.sonic.default.start()
+
+        cls.info("create a Sonic client")
+        cls.client = j.clients.sonic.get('test', host="127.0.0.1", port=1491, password='123456')
+
+    def setUp(self):
+        print('\t')
+        self.info('Test case : {}'.format(self._testMethodName))
+
+        self.data = {
```

Comment:
try to make this data more random,  and save the words u want to query later in a variable